src/Test/Manager/ConnectionManager.php

```diff
@@ -16,6 +16,7 @@ final class ConnectionManager
 {
     /**
      * @psalm-suppress InvalidArgument - Unable to infer createConnection parameters.
+     * @psalm-suppress InternalClass
```

Comment:
The ConnectionFactory at line 34 is marked as Internal.
I suppressed it for now, but we might need to look into a solution?

Reply:
I dont think there really is a good other solution other than copy/pasting it in here and keeping it up to date manually. It's a class that doesn't change often (as in : it's public API), so we'll just deal with changes when they come around.